### ISSUE TITLE
Allow the Nation Core to be Moved After Initial Placement

### ISSUE THREAD

**Issue by connor7000** [mc-ascendancy/ascendancy#157]
From **sprout#6906** on Discord:

"Let the _Chancellor_ [of a Nation] move the Nation Core to a different spot in the same chunk so they don't have to disband and recreate the nation to do so."

**Comment by connor7000** [comment]
Would a feature of this nature be practical given the current implementation of AscendNations, @VolticFroogo?

**Comment by VolticFroogo** [comment]
> Would a feature of this nature be practical given the current implementation of AscendNations, @VolticFroogo?

@connor7000, yes, this should be relatively simple.

**Comment by connor7000** [comment]
Ah! Great news.

Also, @brussels-sprout's comment on Discord isn't hypothetical at all. My very own nation's _Chancellor_ did this the other night!

**Comment by brussels-sprout** [comment]
Yeah I see a lot of people wanting to change their core position that end up disbanding to do it which isn't ideal.

**Comment by VolticFroogo** [comment]
My initial idea is to allow players to claim home blocks, even when they have a home claim, and if they place it in the same chunk as their previous claim, it will replace it.

Does this sound like a decent implementation?

**Comment by brussels-sprout** [comment]
Do you mean giving players the core item and if they place it in the home chunk it becomes the new core and the old one gets removed?

**Comment by VolticFroogo** [comment]
> Do you mean giving players the core item and if they place it in the home chunk it becomes the new core and the old one gets removed?

@brussels-sprout, exactly.

I would just allow them to claim their core block just as if they don't have a home.

**Comment by brussels-sprout** [comment]
Yeah that would work. Is it possible to make it so they can't drop it/put it in a chest/place it outside of the home chunk and remove it from their inventory after some time?

**Comment by VolticFroogo** [comment]
> Yeah that would work. Is it possible to make it so they can't drop it/put it in a chest/place it outside of the home chunk and remove it from their inventory after some time?

@brussels-sprout, this is already the case, home blocks are impossible to transfer between inventories/drop.

**Comment by brussels-sprout** [comment]
Poggers.

**Comment by VolticFroogo** [comment]
Okay, this will be added in a future update of AscendNations.
